.github/workflows/pr.yml

```
@@ -81,10 +81,13 @@ jobs:
           set -x
           git checkout -b branch
           git fetch --unshallow origin main
-          cmd="ansible-test sanity --verbose --docker --color --coverage --failure-ok"
-          $cmd 2>&1 | tee branch.output
+          for version in 3.6 3.7 3.8 3.9 ; do
+            ansible-test sanity --verbose --docker --python $version --color --coverage --failure-ok
+          done 2>&1 | tee branch.output
           git checkout main
-          $cmd > main.output 2>&1
+          for version in 3.6 3.7 3.8 3.9 ; do
+            ansible-test sanity --verbose --docker --python $version --color --coverage --failure-ok
+          done > main.output 2>&1
```

likewise, rather append:
```suggestion
          done >> main.output 2>&1
```

---

started a fix for that at https://github.com/redhatci/ansible-collection-redhatci-ocp/pull/102